README.md

```diff
@@ -30,6 +30,7 @@ current calendars in this repository:
     - [Testing DevEx Team](https://rust-lang.github.io/calendar/testing-devex.ics)
   - [Infrastructure Team](https://rust-lang.github.io/calendar/infra.ics)
   - [Language Team](https://rust-lang.github.io/calendar/lang.ics)
+    - [Spec Team](https://rust-lang.github.io/calendar/spec.ics)
```

Comment:
Normally we'd include t-spec in the t-language calendar's includes when they're nested like this

Reply:
Members of T-lang don't want subteam calendars on the lang team calendar.

Reply:
We've had a similar thing in t-compiler and now have a no sub-team/wg events calendar, I may make that something which the generator does automatically.